INSTALL.rst

```diff
@@ -25,6 +43,8 @@ On all major operating systems, install it via shell/command prompt::
 
   pip install scikit-image
```

Comment:
Might be useful to recommend `python -m pip` everywhere as the way to invoke pip (replace with `python3` depending on the OS and version). That will fail transparently if pip doesn't manage/isn't installed for the interpreter pointed at by `python`.